### PR TITLE
Docs - Added Upgrading PXF Step 2

### PR DESCRIPTION
**Pull request by mperezfuster**
Added Step 2 since it was missing
Also referenced user to the Installation and Configuration Directories to make sure we highlight these changes when upgrading from PXF 5
Preview: https://mireia-pxf-upgrade.sc2-04-pcf1-apps.oc.vmware.com/pxf/6-0/using/upgrade_5_to_6.html

### CODE REVIEW

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 1]
```diff
@@ -35,6 +35,11 @@ Perform this procedure before you upgrade to a new version of PXF:
 
 1. [Install PXF 6.x](installing.html), identify and note the new PXF version number, and then continue your PXF upgrade with [Step 3: Complete the Upgrade to PXF 6.x](#pxfup).
```
Comment (lisakowen) [inline docs/content/upgrade_5_to_6.html.md.erb:36]: i think we can probably remove this line.

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 2]
```diff
@@ -35,6 +35,11 @@ Perform this procedure before you upgrade to a new version of PXF:
...
 
+## <a id="pxfinst"></a>Step 2: Install PXF 6.x
```
Comment (lisakowen) [inline docs/content/upgrade_5_to_6.html.md.erb:38]: Install -> "Installing"

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 3]
```diff
@@ -35,6 +35,11 @@ Perform this procedure before you upgrade to a new version of PXF:
...
+
+1. Follow the steps outlined in [Install PXF 6.x](../release/installing_pxf.html), and identify and note the new PXF version number.
```
Comment (lisakowen) [inline docs/content/upgrade_5_to_6.html.md.erb:40]: Install PXF 6.x -> Installing PXF (this is the name of main topic)

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 4]
```diff
@@ -35,6 +35,11 @@ Perform this procedure before you upgrade to a new version of PXF:
...
+
+1. Check out the new installation layout in [Installation and Configuration Directories](about_pxf_dir.html)
```
Comment (lisakowen) [inline docs/content/upgrade_5_to_6.html.md.erb:42]: lets use the complete topic name:  About the PXF Installation and Configuration Directories.

also, a missing period.

**Review by lisakowen — CHANGES_REQUESTED** [review]

**Review by lisakowen — APPROVED** [review]

File: docs/content/upgrade_5_to_6.html.md.erb [hunk 5]
```diff
@@ -9,7 +9,7 @@ If you have installed, configured, and are using PXF 5.x in your Greenplum Datab
 The PXF upgrade procedure has three steps. You perform one pre-install procedure, the install itself, and then a post-install procedure to upgrade to PXF 6.x:
 
 -   [Step 1: Perform the PXF Pre-Upgrade Actions](#pxfpre)
--   Step 2: Install PXF 6.x
+-   [Step 2: Installing PXF 6.x](#pxfinst)
```
Comment (dyozie) [inline docs/content/upgrade_5_to_6.html.md.erb:12]: Not sure why this was changed from Install to Installing?  These step titles are no longer parallel after this change.

**Review by dyozie — APPROVED** [review]

File: docs/content/upgrade_5_to_6.html.md.erb (hunk 5 above)
Comment (lisakowen) [reply docs/content/upgrade_5_to_6.html.md.erb:12]: my bad.  i thought i was looking at a topic title when i made that comment.  sorry, mireia!

Comment (mperezfuster) [reply docs/content/upgrade_5_to_6.html.md.erb:12]: you were, I just went and changed this one for some reason, it's undone now!

**Pull request merged**